c2cgeoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl

```diff
@@ -1,5 +1,19 @@
 This file includes migration steps for each release of c2cgeoportal.
 
+Version 1.6.12
+==============
+
+1. In the `vars_{{package}}.yaml` file, change:
+
+    smtp_server: smtp.example.com
+
+to
+
+    smtp:
+      host: smtm.example.com
```

Comment:
Use the diff syntax (+/-) a there is done in the rest of the file...

Reply:
Coould you replace **smtm**.example.com by **smtp**.example.com